make/tools.mk

```diff
@@ -260,11 +260,11 @@ OPENOCD_BUILD_DIR := $(DL_DIR)/openocd-build
 
 openocd_install: | $(DL_DIR) $(TOOLS_DIR)
 openocd_install: OPENOCD_URL     := git://git.code.sf.net/p/openocd/code
-openocd_install: OPENOCD_REV     := cf1418e9a85013bbf8dbcc2d2e9985695993d9f4
+openocd_install: OPENOCD_TAG     := v0.9.0
 openocd_install: OPENOCD_OPTIONS := --enable-maintainer-mode --prefix="$(OPENOCD_DIR)" --enable-buspirate --enable-stlink
 
 ifeq ($(OPENOCD_FTDI), yes)
-openocd_install: OPENOCD_OPTIONS := $(OPENOCD_OPTIONS) --enable-ft2232_libftdi
+openocd_install: OPENOCD_OPTIONS := $(OPENOCD_OPTIONS) --enable-ftdi
 endif
 
 ifeq ($(UNAME), Darwin)
```

Comment:
I haven't tested on my Mac yet so it may need a little work. Getting it from brew sounds reasonable, although we'd need to add that as a toolchain pre-req. I like the idea of using brew in general though. :-D